Update Security Findings schema reference: add base_image, reachable-risk fields, severity-override metadata (#37553)
DataDog/documentation: content/en/security/guide/findings-schema.md

{{% /tab %}}
{{< /tabs >}}

> ## Linking to findings

Reviewer: Looks like this section should be restored, per the reviewer note in the PR description:
> ⚠️ Reviewer note: this generated diff also removes the "## Linking to findings" prose section that exists on main. That section is not part of the schema batch. It should be restored before merge (the doc generator does not preserve that manually-added section).

      <td>string</td>
      <td><strong>Path:</strong> <code>@risk_details.is_function_reachable.evidence.not_supported_reason</code><br>Reason why reachability analysis is not supported for this finding. Valid values: <code>language_not_supported</code>, <code>vulnerable_symbol_not_available</code>.</td>
    </tr>
    <tr>
      <td><code>unreachable_at</code></td>
      <td>integer</td>

>       <td><strong>Path:</strong> <code>@risk_details.is_function_reachable.evidence.unreachable_at</code><br>Timestamp in milliseconds (UTC) at which the finding will transition to unreachable if the vulnerable function has not been called.</td>

Reviewer: ```suggestion
      <td><strong>Path:</strong> <code>@risk_details.is_function_reachable.evidence.unreachable_at</code><br>Timestamp in milliseconds (UTC) at which the finding transitions to an unreachable state if the vulnerable function is not called.</td>
```
